UI/Pages/ContactUs.cshtml

```diff
@@ -26,8 +26,7 @@
         </p>
         <h2 class="govuk-heading-m">For issues with your tuition partner </h2>
         <p class="govuk-body">
-            For help on how to handle an issue with your tuition partner, you can <a href="https://www.gov.uk/government/publications/national-tutoring-programme-guidance-for-schools-2022-to-2023" data-testid="read-guidance-link"
-                                                                                     class="govuk-link" target="_blank">read our guidance.</a>
+            For help if you have an issue with your tuition partner, you can  <a asp-page="/ComplaintsPage" class="govuk-footer__link" data-testid="complaints-page-link">read our guidance.</a>                                                                            
```

Comment:
We haven't been consistent on this however please change `asp-page="/ComplaintsPage"` to `asp-page="nameof(ReportIssues)"` so that it will be picked up if the make is renamed or moved.

Reply:
changed